examples/docker-logs.js

```diff
@@ -0,0 +1,19 @@
+import containers from 'k6/x/docker/containers';
+
+export default function () {
+  const containerId = 'YOUR_CONTAINER_ID';
+  const options = {
+    show_stdout: true,
+    // show_stderr: true,
+    // since: '',
+    // until: 10,
+    // timestamp: true,
+    // follow: false,
+    tail: 20,
+    // defaults: true
+  }
+  const result = containers.logs(containerId, options);
```

Comment:
This goes for all of the functions that take an options object: Should we provide some "sane" defaults in this case if a user omits the options object from the call? Or is the docker client already taking care of that?